flink-core/src/main/java/org/apache/flink/configuration/description/BlockElement.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.configuration.description;
+
+/**
+ * Part of description that represents a block e.g. some text, linebreak or a list.
+ */
+public interface BlockElement {
+
+	/**
+	 * Transforms itself into String representation using given format.
+	 *
+	 * @param formatter formatter to use.
+	 * @return string representation
+	 */
+	String format(Formatter formatter);
```

Comment:
This class is effectively identical to `InlineElement`; at the very least this method could be moved into a shared parent class.

Reply:
Added a common parent.